src/index.js

```diff
@@ -2,10 +2,20 @@
 
 const debug = require('debug')
 const log = debug('libp2p:websockets')
-const SWS = require('simple-websocket')
+const SW = require('simple-websocket')
+const isNode = require('detect-node')
+let SWS
+if (isNode) {
+  SWS = require('simple-websocket-server')
```

Comment:
We get some warning of missing deps, namely:

```
WARNING in ./~/ws/lib/Validation.js
Module not found: Error: Can't resolve 'utf-8-validate' in '/Users/david/code/js-libp2p-websockets/node_modules/ws/lib'
 @ ./~/ws/lib/Validation.js 10:19-44

WARNING in ./~/ws/lib/BufferUtil.js
Module not found: Error: Can't resolve 'bufferutil' in '/Users/david/code/js-libp2p-websockets/node_modules/ws/lib'
 @ ./~/ws/lib/BufferUtil.js 10:19-40
```

But since the execution never goes there, no error is caused. Do we need to 'fix' these warnings?